build.fsx

```diff
@@ -280,6 +280,7 @@ let dotnetAssemblyInfos =
       "Fake.Tools.Git", "Running git commands"
       "Fake.Tools.GitVersion", "GitVersion helper"
       "Fake.Tools.Octo", "Octopus Deploy octo.exe tool helper"
+      "Fake.Installer.Squirrel", "Squirrel for windows Squirrel.exe tool helper"
```

Comment:
This is sorted alphabetically so can you move it to the correct place ;).
I started sorting it once I stopped finding anything :D

Reply:
O come after S it's sorted alphabetically with the previous name :)